spacetemplate/assets/legacy.yaml

```diff
@@ -15,14 +15,17 @@ work_item_types:
   extends: "86af5178-9b41-469b-9096-57e5155c3f31"
   name: Task
   can_construct: yes
-  description: TBD
+  description: Work assigned to various team members to implement a feature.
```

Comment:
For multi-line strings please use `>` and indentation as done in the space template's description.

Reply:
Fixed

Reply:
Will keep this in mind next time. Thank you.